### PR TITLE
[AETHER-1265] Integrate Java profiler in TOST env

### PR DESCRIPTION
**Pull request by pierventre**
Add a nodeport for each atomix instance to enable the connection between client and profiler agent.

### CODE REVIEW

File: atomix/templates/nodeports.yaml
Comment (charlesmcchan) [inline atomix/templates/nodeports.yaml]: See my comment in ONOS helm chart PR #407. I'd suggest renaming this to
`Values.profiler.nodePort` to avoid confusion.

**Review by kuujo — APPROVED** [review]
@charlesmcchan I'm good with this, but I'll wait for your approval of the value name change

**Pull request merged**
